### PR TITLE
Document usage of kubecontexts in subctl benchmark

### PR DESCRIPTION
**Pull request by maayanf24**
Signed-off-by: Maayan Friedman <maafried@redhat.com>

### CODE REVIEW

**Review by skitt — APPROVED** [review]

**Review by nyechiel — APPROVED** [review]

File: src/content/operations/deployment/subctl/_index.en.md [hunk 1]
```diff
@@ -252,6 +252,7 @@ When running `benchmark latency`, two types of tests will be executed:
 |:------------------------------------|:----------------------------------------------------------------------------|
 | `--intra-cluster`                   | Performs the benchmark test within a single cluster between Pods from a Non-Gateway node to a Gateway node
 | `--verbose`                         | Produce verbose logs during benchmark tests
+| `--kubeconfig` `<string>`           | Absolute path(s) to the kubeconfig file(s)
```
Comment (tpantelis) [inline src/content/operations/deployment/subctl/_index.en.md]: It seems we should also document `--kubecontexts  | comma-separated list of two kubeconfig contexts to use.`

File: src/content/operations/deployment/subctl/_index.en.md [hunk 2]
```diff
@@ -226,7 +226,7 @@ that this verification is disruptive.
 
 #### `benchmark throughput`
 
-`subctl benchmark throughput <kubeconfig1> [<kubeconfig2>] [flags]`
+`subctl benchmark throughput --kubecontexts <context1>[,<context2>] [flags]`
```
Comment (tpantelis) [inline src/content/operations/deployment/subctl/_index.en.md:229]: The command requires two kubecontexts.

```suggestion
`subctl benchmark throughput --kubecontexts <context1>,<context2> [flags]`
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: src/content/operations/deployment/subctl/_index.en.md (hunk 2 above)
Comment (maayanf24) [reply src/content/operations/deployment/subctl/_index.en.md:229]: when running with `--intra-cluster` then only one kubecontext is needed

File: src/content/operations/deployment/subctl/_index.en.md (hunk 1 above)
Comment (maayanf24) [reply src/content/operations/deployment/subctl/_index.en.md]: I followed the same pattern as in `subctl verify`, in both cases using `--kubecotexts` is not optional and documented as part of the command usage

Comment (tpantelis) [reply src/content/operations/deployment/subctl/_index.en.md]: Ok. For `verify`, `--kubeconfig` is not documented so perhaps we shouldn't here as well for consistency?

**Review by tpantelis — APPROVED** [review]

**Pull request merged**